packages/http-server/src/server.ts

```diff
@@ -119,7 +104,8 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
           if (output.headers) {
             reply.headers(output.headers);
           }
-          reply.send(output.body);
+
+          reply.send(serialize(output.body, reply.getHeader('content-type')));
```

Comment:
If possible let's try to keep serialisation and sending separated; this will allow Fastify to give us more precise errors in case something goes wrong.

```suggestion
          reply
            .serializer((payload: unknown) => serialize(payload, reply.getHeader('content-type')))
            .send(output.body);
```

Reply:
Done 👍